incl/misc/getTopArtists.php

```diff
@@ -26,35 +27,41 @@
 	$robsult = curl_exec($ch);
 	curl_close($ch);
 	echo $robsult;
-} else {
+}
+if($redirect == 0) {
```

Comment:
I don't think it matters if $redirect is 0 or 2, it should just execute a different result